src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/CodeBasedDependencyAlgorithm.cs

```diff
@@ -67,6 +67,15 @@ public static void AddDependenciesDueToMethodCodePresence(ref DependencyList dep
                         "UniversalCanon signature of method"));
                 }
 
+                if (method.IsVirtual && !method.HasInstantiation)
```

Comment:
We were having a chat about this with @nattress, and you were suggesting we it made more sense to have it in CodeBasedDependencyAlgorithm.cs, that's why he put it there in his bug workaround (https://github.com/nattress/corert/tree/Tizen)

I can't remember the details of this discussion... Do you remember?
It does make sense for me to move it to GetVirtualInvokeMapDependencies, I just can't remember if there was any reason not to.